buildscripts/checkstyle.xml

```diff
@@ -157,14 +157,15 @@
              <message key="ws.notPreceded"
              value="GenericWhitespace ''{0}'' is not preceded with whitespace."/>
         </module>
-        <module name="Indentation">
-            <property name="basicOffset" value="2"/>
-            <property name="braceAdjustment" value="0"/>
-            <property name="caseIndent" value="2"/>
-            <property name="throwsIndent" value="4"/>
-            <property name="lineWrappingIndentation" value="4"/>
-            <property name="arrayInitIndent" value="2"/>
-        </module>
+        <!-- <!-\- Checkstyle indentation rules conflict with google-java-format: -\-> -->
```

Comment:
Do you want to remove these lines?

Reply:
I thought these checks were from Checkstyle's google_checks.xml, so it would be better to comment them out to show what we changed.  It would also help us update the file with new changes from Checkstyle.

Reply:
SGTM